docs/source/algorithms/Fit-v1.rst

```diff
@@ -84,6 +84,19 @@ Browser <http://www.mantidproject.org/MantidPlot:_Data_Analysis_and_Curve_Fittin
 which allows all the settings to be specified via its graphical user
 interface.
 
+The confidence bands (CB) on the calculated fit curve (:math:`\mathbf{\hat{y}}`) are obtained using the method outlined in `kmpfit`.
+For a function, :math:`f(\mathbf{p})`, with fit parameters :math:`\mathbf{p}`, the confidence interval is given by,
+
+.. math:: CB=\mathbf{f}(\mathbf{p}) \pm \mathbf{\sigma_f}
+
+with :math:`\mathbf{\sigma_f}` defined as,
+
+.. math:: \mathbf{\sigma_f}^2= \chi_{\nu}^2 \sum_{j=0}^{j=n}\sum_{k=0}^{k=n}\frac{\partial f}{\partial p_j}\frac{\partial f}{\partial p_k}\, \mathbf{C}_{jk}
+
+where :math:`\mathbf{C}_{jk}` is the covriance matrix and :math:`\chi_{\nu}^2` the reduced chi squared value.
+This interval defines the region where there is a 68.3% chance to find the true value of :math:`f(\mathbf{p})`.
```

Comment:
isn't 68.3% just two sigma?

Reply:
68.3 would be 1 sigma right, I just chose to put it as a truncated percent as its easier to get a feel for the significance level

Reply:
it's +/- 1 sigma so you could indeed call it 2 sigma but my understanding is that is not the common terminology

Reply:
It was more a question.